scripts/extract_timings.py

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env python
+# Time-stamp: <2016-05-31 14:58:27 chrplr>
+#
+
+""" Extracts slice timing information from a dicom file.
+To be passed  as argument to slice_order for slice timing correction algorithms """
+
```

Comment:
If I remember good, we were considering hosting a small dataset (say on NTRC), that could be used here to make this demo self-contained. Any update ?